lib/util.js

```diff
@@ -9,6 +9,12 @@ exports.URLSafeUUID = function() {
   return URLSafeBase64.encode(uuid.v4(null, new Buffer(16)));
 };
 
```

Comment:
I'm going to change this to a 6 digit pin. Also, this number needs to be converted to a string, since the hash is a string. Otherwise we will get type errors.